PendingReleaseNotes.md

```diff
@@ -20,8 +20,10 @@
 - PriorityClassNames can now be added to the Rook/Ceph components to influence the scheduler's pod preemption.
   - mgr/mon/osd/rbdmirror: [priority class names configuration settings](Documentation/ceph-cluster-crd.md#priority-class-names-configuration-settings)
   - filesystem: [metadata server settings](Documentation/ceph-filesystem-crd.md#metadata-server-settings)
-  - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)    
+  - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)
   - nfs: [samples](Documentation/ceph-nfs-crd.md#samples)
+- New CR property available in the Operator: `ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE`(5 seconds by default). Represents the time to wait until the node controller will move Rook pods to other nodes after detecting an unreachable node. Pods affected by this setting are: mgr, rbd, mds, rgw, nfs, PVC based mons and osds, and ceph toolbox. The value used in this variable replaces the default value of 300 seconds added automatically by k8s as Pod Toleration for `node.kubernetes.io/unreachable`.
```

Comment:
@jmolmo Why not call this `ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS`? Then it would match the K8s setting `UNREACHABLE_NODE_TOLERATION_SECONDS`. We don't really need the flexibility for other units besides seconds. It's just a minor simplification, but I agree with @BlaineEXE that it would be more readable.

Reply:
Changed! :-)